### ISSUE TITLE
When *10000 speed button pressed, game crashes

### ISSUE THREAD

**Issue by hermonochy** [pioneerspacesim/pioneer#5831]
When the button for *10000 speed is clicked after leaving hyperspace, the entire game crashes, giving this error message:`Segmentation fault (core dumped)
`

**Comment by impaktor** [comment]
Is it reproducible?
Are you on 2024-03-14 version?

**Comment by hermonochy** [comment]
Yes, I am on pioneer 2024-03-14.This error also comes up when trying to dock at a spaceport, but space stations work.

**Comment by impaktor** [comment]
I don't think this will go anywhere unless we get a [backtrace](https://wiki.pioneerspacesim.net/wiki/FAQ#Debug_backtrace), or it can be reproduced, (and backtraced).

Clicking time accel after leaving hyperspace is a pretty standard thing to do, that seems to work 99% (100%?) of the time for others.
